appveyor.yml

```diff
@@ -50,6 +49,8 @@ build_script:
   - msbuild samples\client\petstore\csharp\OpenAPIClientNetStandard\Org.OpenAPITools.sln /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
   # install openapi-generator locally
   - mvn clean install --quiet
+  # run the locally installed openapi-generator-gradle-plugin
+  - gradle -PopenApiGeneratorVersion=4.0.2-SNAPSHOT -b modules\openapi-generator-gradle-plugin\samples\local-spec\build.gradle buildGoSdk --info
```

Comment:
I think `-PopenApiGeneratorVersion=4.0.2-SNAPSHOT` can be removed after changes to fix `release_version_update.sh` and `release_version_update_docs.sh`